### PR TITLE
Closes #1197 xASL_module_Population: add x.S.DataTypes

### LINKED ISSUE #1197: Iterate Statistics over multiple DataTypes (E.g., TExch, SD, etc)
### Description

If we had another string to line 230 (see printscreen below), we need to iterate the ROIstatistics through the measures as well (so not only qCBF but also TExch, SD, etc)
image

### Tasks

- [x] Check if x.S.InputDataStr can be a cell array
- [x] Add this option to dataPar.json, so that we don't need to hardcode everytime we need the stats for TExch e.g.

### Release notes
Allow configuring the image contrast for statistics in dataPar.json.

### PR DESCRIPTION
**Pull request by HenkMutsaerts**
### Linked issue

Closes #1197

### How to test

- [x] Add "x.S.DataTypes": ["qCBF", "Texch"] to `dataPar.json` and run the population module @BeatrizPadrela to test if this new feature works
- [ ] Run the population module without this new setting to see if the default still works (it should default to `qCBF`as it originally did)

### Comments

@jan-petr Should we add this new parameter to the `dataParTemplate.md` (this is now removed and only online right?)

### CODE REVIEW

File: Modules/xASL_module_Population.m [hunk 1]
```diff
@@ -201,75 +201,84 @@
...
-    % x.S.InputAtlasPath            = fullfile(x.D.MapsSPMmodifiedDir,'WholeBrain.nii');
-    % % x.S.InputAtlasPath            = fullfile(x.D.AtlasDir,'HOcort_CONN.nii');
-    % xASL_wrp_GetROIstatistics( x);
-    % x.S.InputDataStr              = 'SNR'; % 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
-    % x.S.InputAtlasPath            = fullfile(x.D.MapsSPMmodifiedDir,'WholeBrain.nii');
-    % xASL_wrp_GetROIstatistics( x);
-
-    x.S.InputDataStr = 'qCBF'; % 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
-	x.S.InputDataStrNative = 'CBF'; % 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
+    x.S.InputDataStr = 'qCBF'; % 'Texch' 'ATT' 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
```
Comment (jan-petr) [inline Modules/xASL_module_Population.m:227]: These lines can be probably commented out. This value will be always overwritten.

File: Modules/xASL_module_Population.m [hunk 2]
```diff
@@ -201,75 +201,84 @@
...
     
-    % Default & Fallback for Atlases
+    % Default atlases/ROIs
     if ~isfield(x.S,'Atlases')
         x.S.Atlases = {'TotalGM','DeepWM'}; % Default
     end
-
+    
+    % Default datatypes
+    if ~isfield(x.S,'DataTypes')
```
Comment (jan-petr) [inline Modules/xASL_module_Population.m:210]: Probably add
|| isempty(x.S.DataTypes)
just to be sure.

**Review by jan-petr — CHANGES_REQUESTED** [review]
Two minor things.
Also, we should add this new dataPar parameter to the Documentation.
Can you please draft something and I'll put it here?
https://exploreasl.github.io/Documentation/1.10.0beta/ProcessingParameters/

File: Modules/xASL_module_Population.m (hunk 2 above)
Comment (HenkMutsaerts) [reply Modules/xASL_module_Population.m]: Done

File: Modules/xASL_module_Population.m (hunk 1 above)
Comment (HenkMutsaerts) [reply Modules/xASL_module_Population.m]: Done

**Comment by HenkMutsaerts** [comment]
Can you re-review?

For the dataPar template online:

x.S.[...]

DataTypes = Vector of cells specifying which images to take data from, in the ROI analysis. E.g., {'Tex’, 'ATT' 'SD' 'M0' ‘rc1T1’}

OPTIONAL, DEFAULT = {‘qCBF’}

**Review by jan-petr — APPROVED** [review]
OK

**Pull request merged**